GitUI/CommandsDialogs/SettingsDialog/Pages/ScriptsSettingsPage.cs

```diff
@@ -17,7 +17,10 @@ public partial class ScriptsSettingsPage : SettingsPageWithHeader
         #region translation
 
         private readonly TranslationString _scriptSettingsPageHelpDisplayArgumentsHelp = new TranslationString("Arguments help");
-        private readonly TranslationString _scriptSettingsPageHelpDisplayContent = new TranslationString(@"User Input:
+        private readonly TranslationString _scriptSettingsPageHelpDisplayContent = new TranslationString(@"Use {option} for normal replacement.
+Use {{option}} for qouted replacement.
```

Comment:
"qouted" => "quoted"

Reply:
![image](https://user-images.githubusercontent.com/3629489/50510370-6a198280-0a57-11e9-9aab-0c2cf8a30da4.png)

very helpful feature

```suggestion
Use {{option}} for quoted replacement.
```

Reply:
Indeed, but on smartphone, that's a little harder 😉

Reply:
I sometimes use this app.  https://play.google.com/store/apps/details?id=com.github.pockethub.android